app/src/main/java/com/codesoom/assignment/HttpStatus.java

```diff
@@ -0,0 +1,19 @@
+package com.codesoom.assignment;
+
+public enum HttpStatus {
+    Ok(200),
+    Created(201),
+    NoContent(204),
+    NotFound(404),
+    InternalServerError(500);
```

Comment:
Java에서는 일반적으로 `enum`은 대문자와 _ 를 사용해 이름을 만들어 줍니다. 이런건 규칙으로 강제되지는 않지만 전 세계 자바 개발자들의 암묵적인 약속이죠.

```suggestion
    OK(200),
    CREATED(201),
    NO_CONTENT(204),
    NOT_FOUND(404),
    INTERNAL_SERVER_ERROR(500);
```